File: ListingManager.Domain/Model/OpenHouse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ListingManager.Domain.Model
{
    public class OpenHouse
    {
        public int OpenHouseId { get; set; }
        public DateTime OpenHouseBeginDate { get; set; }
        public DateTime OpenHouseEndDate { get; set; }
        public int ListingId { get; set; }
        public virtual Listing Listing { get; set; }
    }
}

File: ListingManager.Web/listing-app/src/app/models/openhouse.ts
import { Listing } from 'app/models/listing'

export class OpenHouse {
    constructor(
        public OpenHouseId?: number,
        public OpenHouseBeginDate?: Date,
        public OpenHouseEndDate?: Date,
        public ListingId?:number,
        public Listing?: Listing[],
    ) { }
}